mintpy/closure_phase_bias.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+############################################################
+# Program is part of MintPy                                #
+# Copyright (c) 2013, Zhang Yunjun, Heresh Fattahi         #
+# Author: Yujie Zheng, Feb 2022                            #
+############################################################
+# Compute average con-nl closure phase and output mask identifying areas suseptible to closure phase errors.
+
+import os, sys
+import h5py
+from mintpy.objects import ifgramStack
+import numpy as np
+from mintpy.utils import readfile, writefile, ptime, utils as ut
+from mintpy import ifgram_inversion as ifginv
+import argparse
+import matplotlib.pyplot as plt
+
+################################################################################
+REFERENCE = """reference:
+  Zheng, Y., et al., (2022) On Closure Phase and Systematic Bias in Multilooked SAR Interferometry, IEEE TGRS, under review (minor revision)
+"""
+EXAMPLE = """example:
+  closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20
+"""
+
+def create_parser():
+    parser = argparse.ArgumentParser(description = 'Create an indication map for closure phase bias.')
+    parser.add_argument('-i','--ifgramstack',type = str, dest = 'ifgram_stack',help = 'interferogram stack file that contains the unwrapped phases')
+    parser.add_argument('--nl', dest = 'nl', type = int, default = 20, help = 'connection level that we are correcting to (or consider as no bias)')
+    parser.add_argument('--theta',dest = 'theta', type = float, help = 'Threashold for phase (unit: radian), default to be 3 sigma')
```

Comment:
Would you please complete the help for this parameter. In particular "3 sigma of what distribution"? 
I know you have a nice comment written [here](https://github.com/insarlab/MintPy/pull/740/files#diff-8c628b98ef4d92190283a31ccf1098be186400827ef61e2deeabb1ffa3c07c17R137). Thank you for that. However, it would be nice if the user would have a better idea of this threshold from help message.

Reply:
Thanks for the suggestion. I have changed the input from a phase value (unit radian), to a number of sigma (number of standard deviation for a Gaussian distribution). I have updated the help message and the code accordingly.